src/utils/apiCalls.js

```diff
@@ -42,6 +42,19 @@ const getMessagesAfterId = async (channelId, limit, afterId, addFirstMessage = f
     return messages;
 }
 
+//returns an array of servers this bot is in
+const getServers = async () => {
+    return await getAPICall(`${baseUrl}/users/@me/guilds`)
+}
```

Comment:
`await` is unnecessary here (as is `async`, technically).
```suggestion
const getServers = () => {
    return getAPICall(`${baseUrl}/users/@me/guilds`);
}
```